pkgs/development/python-modules/opentelemetry-instrumentation-botocore/default.nix

```
@@ -0,0 +1,56 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  hatchling,
+  pythonOlder,
+  opentelemetry-api,
+  opentelemetry-instrumentation,
+  opentelemetry-semantic-conventions,
+  botocore,
+  markupsafe,
+  moto,
+  opentelemetry-instrumentation-botocore,
+  opentelemetry-test-utils,
+  opentelemetry-propagator-aws-xray,
+}:
+
+buildPythonPackage rec {
+  inherit (opentelemetry-instrumentation) version src;
+  pname = "opentelemetry-instrumentation-botocore";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.8";
+
+  sourceRoot = "${opentelemetry-instrumentation.src.name}/instrumentation/opentelemetry-instrumentation-botocore";
+
+  build-system = [ hatchling ];
+
+  dependencies = [
+    opentelemetry-api
+    opentelemetry-instrumentation
+    opentelemetry-semantic-conventions
+    opentelemetry-propagator-aws-xray
+  ];
+
+  nativeCheckInputs = [ botocore ];
```

```suggestion
  nativeCheckInputs = [
    botocore
    pytestCheckHook
  ];
```
Could you add `pytestCheckHook` to set the checkPhase?